lang/src/address_lookup_table_program.rs

```diff
@@ -0,0 +1,39 @@
+use crate::prelude::*;
+use crate::solana_program::address_lookup_table;
+use solana_program::pubkey::Pubkey;
+use solana_program::address_lookup_table::state::AddressLookupTable as SolanaAddressLookupTable;
+
+#[derive(Debug, Clone)]
+pub struct AddressLookupTable;
+impl anchor_lang::Id for AddressLookupTable {
+    fn id() -> Pubkey {
+        address_lookup_table::program::ID
+    }
+}
+
+#[derive(Clone, shrinkwraprs::Shrinkwrap)]
```

Comment:
After removing the `shrinkwraprs::Shrinkwrap` derivation, could you add `Deref` impl? For example:

https://github.com/coral-xyz/anchor/blob/6df05aa5b5d9971041e3d25a246d935252b6e576/spl/src/token.rs#L469-L475

```suggestion
#[derive(Clone)]
```